menu.sh

```diff
@@ -140,6 +143,9 @@ function yml_builder() {
 	#if an env file exists check for timezone
 	[ -f "./services/$1/$1.env" ] && timezones ./services/$1/$1.env
 
+    # if a volumes.yml exists, append to overall volumes.yml file
+    [ -f "./services/$1/volumes.yml" ] && cat "./services/$1/volumes.yml" >> docker-volumes.yml
```

Comment:
This will likely need to be updated in the future once we have yaml merging on. Just letting you know in case it breaks your current setup when that upgrade happens.

Reply:
Ok no problem. I did in this way to have the code written in a similar fashion to the original